wandb/integration/catboost/catboost.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+"""
+catboost init
+"""
+
+import wandb
+
+
+class WandbCallback:
+    """`WandbCallback` automatically integrates CatBoost with wandb.
+
+    Arguments:
+        - metric_period: (int) if you are passing `metric_period` to your CatBoost model please pass the same value here (default=1).
+
+    Passing `WandbCallback` to CatBoost will:
+    - log training and validation metrics at every `metric_period`
+    - log iteration at every `metric_period`
+
+    Example:
+        ```
+        train_pool = Pool(train[features], label=train['label'], cat_features=cat_features)
+        test_pool = Pool(test[features], label=test['label'], cat_features=cat_features)
+
+        model = CatBoostRegressor(iterations=100,
+            loss_function='Cox',
+            eval_metric='Cox',
+        )
+
+        model.fit(train_pool,
```

Comment:
formatting, lets keep this consistent with the open bracket / close bracket styling in `model = CatBoostRegressor...`